recipes/qredis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "qredis" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qredis-{{ version }}.tar.gz
+  sha256: 67b1e11e106ff8e68418b25ff06e55d772f529f94c13197927b63fdd299d51b9
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - qredis=qredis.window:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - msgpack-python
+    - msgpack-numpy
+    - qtpy
+    - pyqt =5
```

Comment:
Any reason to not use `>=5` ?

Reply:
Yes, qredis direct dependency is qtpy. So far qtpy only supports PyQt4 or PyQt5 but not PyQt6.
Would it make more sense to put `<6` instead ?

Reply:
That would probably be more correct in the face of `only supports PyQt4 or PyQt5`, but I guess `pyqt =5` works just fine

Also making a comment https://github.com/spyder-ide/qtpy/issues/233 here could help ;-)

Reply:
Sorry for the silence. I have no excuse.
Looks like qtpy>=2 added support for PyQt6 so I have just updated `meta.yaml`.